google/cloud/firestore_v1/base_vector_query.py

```diff
@@ -45,6 +45,8 @@ def __init__(self, nested_query) -> None:
         self._query_vector: Optional[Vector] = None
         self._limit: Optional[int] = None
         self._distance_measure: Optional[DistanceMeasure] = None
+        self._distance_result_field: Optional[str] = None
+        self._distance_threshold: Optional[Union[int, float]] = None
```

Comment:
nit: `float` implies `Union[int, float]` when doing type annotations: https://stackoverflow.com/questions/50928592/mypy-type-hint-unionfloat-int-is-there-a-number-type

Reply:
Done.